pkg/auth/dependency/oidc/metadata.go

```diff
@@ -1,13 +1,24 @@
 package oidc
 
-type MetadataProvider struct{}
+import "github.com/skygeario/skygear-server/pkg/auth/dependency/urlprefix"
+
+type MetadataProvider struct {
+	URLPrefix urlprefix.Provider
+}
 
 func (p *MetadataProvider) PopulateMetadata(meta map[string]interface{}) {
+	meta["issuer"] = p.URLPrefix.Value().String()
 	meta["scopes_supported"] = AllowedScopes
 	meta["subject_types_supported"] = []string{"public"}
 	meta["id_token_signing_alg_values_supported"] = []string{"RS256"}
+	meta["claims_supported"] = []string{
+		"iss",
+		"aud",
+		"iat",
+		"exp",
+		"sub",
```

Comment:
I think `nonce` is not exactly a claim of ID token, since it is only present iif authorization request has nonce. Other case, including refreshing existing tokens, would not return a ID token containing nonce. Also, Google's OIDC discovery document also does not have it, yet seems to support nonce.